proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/organization.proto";
+import "buf/alpha/registry/v1alpha1/plugin.proto";
+import "buf/alpha/registry/v1alpha1/repository.proto";
+import "buf/alpha/registry/v1alpha1/user.proto";
+
+// UserService is the User service.
+service AdminService {
+  // ForceDeleteUser forces to delete a user. Resources and organizations that are
+  // solely owned by the user will also be deleted.
+  rpc ForceDeleteUser(ForceDeleteUserRequest) returns (ForceDeleteUserResponse);
```

Comment:
I would rather see a generic "DeleteUser" with a parameter around deleting. More flexible if for whatever reason we want to let admins to "soft"(?) deletes of other users.

Reply:
Resolved offline, keeping as is.